gooddata-sdk/gooddata_sdk/catalog/base.py

```diff
@@ -40,10 +40,9 @@ def from_dict(cls: Type[T], data: Dict[str, Any], camel_case: bool = True) -> T:
         Creates object from dictionary. It needs to be specified if the dictionary is in camelCase or snake_case.
         """
         # This branch does not have to exist, but in case of camel_case=False it can perform fewer calls
```

Comment:
This comment no longer makes sense I think

Reply:
Thanks. Removed.